bio/jannovar/meta.yaml

```
@@ -1,4 +1,4 @@
 name: "jannovar"
-description: Annotate predicted effect of nucleotide changes with `Jannovar<https://doc-openbio.readthedocs.io/projects/jannovar/en/master/>`_
+description: Annotate predicted effect of nucleotide changes with `Jannovar <https://doc-openbio.readthedocs.io/projects/jannovar/en/master/>`_
```

It might be better to just set the URL, no?
```suggestion
description: Annotate predicted effect of nucleotide changes with Jannovar
url: https://doc-openbio.readthedocs.io/projects/jannovar/en/master/
```